plugins/modules/idrac_network_attributes.py

```diff
@@ -0,0 +1,534 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#
+# Dell OpenManage Ansible Modules
+# Version 8.4.0
+# Copyright (C) 2023 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+---
+module: idrac_network_attributes
+short_description: Configures the iDRAC network attributes
+version_added: "8.4.0"
+description:
+  - This module allows to configure iDRAC network settings.
+extends_documentation_fragment:
+  - dellemc.openmanage.idrac_auth_options
+options:
+  network_adapter_id:
+    type: str
+    required: true
+    description:
+        - FQDD of the network adapter device that represents the physical network adapter capable of connecting to a computer network.
+        - An example of FQDD of the network adapter is 'NIC.Mezzanine.1A'
+  network_device_function_id:
+    type: str
+    required: true
+    description:
+      - FQDD of the network adapter device function that represents a logical interface exposed by the network adapter.
+      - An example of FQDD of the network adapter device function is 'NIC.Mezzanine.1A-1-1'
+  network_attributes:
+    type: dict
+    description:
+      - "Dictionary of network attributes and value. To view the list of attributes and its structure, see the below API
+        U(https://I(idrac_ip)/redfish/v1/Systems/System.Embedded.1/NetworkAdapters/<network_id>/NetworkDeviceFunctions/
+        <network_port_id>/Settings)."
+      - I(network_attributes) is mutually exclusive with I(oem_network_attributes).
+  oem_network_attributes:
+    type: dict
+    description:
+      - "The attributes must be part of the Integrated Dell Remote Access Controller Attribute Registry.
+        To view the list of attributes in Attribute Registry for iDRAC9 and newer versions. For more information,
+        see, U(https://I(idrac_ip)/redfish/v1/Chassis/System.Embedded.1/NetworkAdapters/<network_id>/NetworkDeviceFunctions/
+        <network_port_id>/Oem/Dell/DellNetworkAttributes/<network_port_id>)
+        and U(https://I(idrac_ip)/redfish/v1/Registries/NetworkAttributesRegistry_<network_port_id>/
+        NetworkAttributesRegistry_network_port_id.json)."
+      - For iDRAC8 based servers, derive the network attribute name from Server Configuration Profile.
+      - I(oem_network_attributes) is mutually exclusive with I(network_attributes).
+  resource_id:
+    type: str
+    description:
+      - Id of the resource.
+      - If the value for resource ID is not provided, the module picks the first resource ID available from the list of system resources returned by the iDRAC.
```

Comment:
Spaces should be inserted to align. Might get sanity issue.

Reply:
There's no sanity issue, will attach sceenshot.